### DISCUSSION TITLE
Unhandled exception caught during execution: unable to create directories '' for write

### DISCUSSION THREAD

**Discussion by conghs08** [modular/modular#1552] [Q&A]
excuse me for asking such a question, but simple code, read is ok, but can not write!

output:
mojo rw.mojo 
6 4

Unhandled exception caught during execution: unable to create directories '' for write
mojo: error: execution exited with a non-zero result: 1

code:
def main():

    var  f = open("data.in", "r")

    let rr : String = f.read()
    print(rr)

    var fout = open("by_mojo.out", "w")
    fout.write(str("result") + '\n')
    fout.close()

    f.close()

**Comment by mstrechen** [comment]
@conghs08 I encountered the same problem. You could use `"./by_mojo.out"` instead of `"by_mojo.out"`, this should work. 

I assume there is a bug in implementation which tries to find a folder to create (for cases when you want to create a file in not-yet-existing subfolder), so I would suggest reporting this as a bug.
